layouts/partials/navigation.html

```diff
@@ -1,95 +1,131 @@
+        <ul class="navbar-nav ml-auto">
+          {{ range site.Menus.main }}
+          {{ if .HasChildren }}
+          <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle text-white" href="#" role="button" data-toggle="dropdown"
+              aria-haspopup="true" aria-expanded="false">
+              {{ .Name }}
+            </a>
+            <div class="dropdown-menu">
+              {{ range .Children }}
+              <a class="dropdown-item" href="{{ .URL | absLangURL }}">{{ .Name }}</a>
+              {{ end }}
+            </div>
+          </li>
+          {{ else }}
+          <li class="nav-item">
+            <a class="nav-link text-white" href="{{ .URL | absLangURL }}">{{ .Name }}</a>
+          </li>
+          {{ end }}
+          {{ end }}
+        </ul>
+        <!-- Language List -->
+        {{- if site.IsMultiLingual }}
+        <select class="lang-list bg-primary {{ if not .IsHome }}mb-3 mb-md-0{{ end }}" id="select-language"
+          onchange="location = this.value;">
+          {{ $siteLanguages := site.Languages}}
+          {{ $pageLang := .Page.Lang}}
+          {{ range .Page.AllTranslations }}
+          {{ $translation := .}}
+          {{ range $siteLanguages }}
+          {{ if eq $translation.Lang .Lang }}
+          {{ $selected := false }}
+          {{ if eq $pageLang .Lang}}
+          <option id="{{ $translation.Language }}" value="{{ $translation.Permalink }}" selected>{{ .LanguageName }}
+          </option>
+          {{ else }}
+          <option id="{{ $translation.Language }}" value="{{ $translation.Permalink }}">{{ .LanguageName }}</option>
+          {{ end }}
+          {{ end }}
+          {{ end }}
+          {{ end }}
+        </select>
         {{ end }}
-      </select>
-      {{ end }}
-      <p class="text-white unicef-glob ml-4 mt-3">Visit <a href="{{ $orgUrl }}" class="text-white">{{ $orgName }} <i
-        class="fas fa-angle-double-right"></i></a></p>
+        <p class="text-white unicef-glob ml-4 mt-3">Visit <a href="{{ $orgUrl }}" class="text-white">{{ $orgName }} <i
+              class="fas fa-angle-double-right"></i></a></p>
+      </div>
     </div>
-  </div>
-  {{ if not .IsHome }}
-  <div class="w-100 py-2 breadcrumbBar text-light" style=" {{ if .IsHome }}background-color: transparent{{ else }}background-color: {{ site.Params.text_color_dark }}{{ end }}">  
-    <div class="container">
-      {{ partial "breadcrumb.html" . }}
-      <a data-toggle="collapse" data-target="#sidebarMobile"  aria-controls="navigation" aria-expanded="false" aria-label="Toggle navigation"> <i class="ti-menu text-light sidebarIcon"></i></a>
-      <div class="text-light text-center sidebarMobile collapse justify-content-center" id="sidebarMobile">
-        <div class="sidebarBox">
-          <ul>
-            {{ $currentNode := . }}
-            {{range .Site.Home.Sections.ByWeight}}
+    {{ if not .IsHome }}
+    {{ end }}
+  </nav>
+  <nav id="breadcrumbNav">
```

Comment:
This is not valid HTML. There should only be one `<nav>` tag in the DOM. Instead, keep a single `<nav>` on the start and end of this file, and use `<div>` tags to create the styling you need.